cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -227,7 +228,7 @@ func (scaleSet *ScaleSet) getCurSize() (int64, error) {
 		set, rerr = scaleSet.manager.azClient.virtualMachineScaleSetsClient.Get(ctx, scaleSet.manager.config.ResourceGroup, scaleSet.Name)
 		if rerr != nil {
 			klog.Errorf("failed to get information for VMSS: %s, error: %v", scaleSet.Name, rerr)
-			return -1, err
+			return -1, rerr.Error()
```

Comment:
Open to discussion: I updated this to `rerr.Error()`, because I feel like we should be returning the error from the compute RP rather than the `err` from the `getVMSSFromCache` call.
cc @comtalyst who made this `rerr` update during cloud-provider-azure refactor.